spec/acceptance/suites/default/10_rsyslog_integration_spec.rb

```diff
@@ -232,39 +232,65 @@ def restart_rsyslog(hosts)
       end
     end
 
-    it 'should generate a local yum log' do
-      hosts.each do |host|
+    hosts.each do |host|
+      # DNF systems don't log to yum.log
+      next if host.which('dnf')
+
+      it 'should generate a local yum log' do
+        skip "#{host} uses 'dnf'" if host.which('dnf')
+
         host.install_package('expect')
         unless host.host_hash[:roles].include?('syslog_server')
           on(host, "grep 'Installed: expect' /var/log/yum.log")
         end
       end
-    end
 
-    it 'should forward yum logs' do
-      begin
-        verify_remote_log_messages(['Installed: expect'], 'secure.log', hosts, domain)
-      rescue Beaker::Host::CommandFailure => e
-        skip "#{self.class.description} failed => #{e}"
+      it 'should forward yum logs' do
+        skip "#{host} uses 'dnf'" if host.which('dnf')
+
+        begin
+          verify_remote_log_messages(['Installed: expect'], 'secure.log', host, domain)
```

Comment:
See note about `dnf.rpm.log` from above; we should probably be forwarding this log for EL8, correct?

Reply:
As with the note above, looks like `dnf` does not have configuration for logging to syslog; so if we want to handle this information with rsyslog, we'll need to update the profile for `simp_rsyslog` to include an `imfile` for the `dnf.rpm.log` file or something to send these messages over. This should be noted in backlog for both projects and resolved later.